### DISCUSSION TITLE
HERO12 Labs update 1.12.70 release

### DISCUSSION THREAD

**Discussion by dnewman-gpsw** [gopro/labs#570] [Announcements]
This is a bug fix release for the main firmware and Labs

MAIN FW: BUG FIXES
- Fixes bug where audio is not captured with the Media Mod microphones when used with Bluetooth audio.
- Fixes bug where photos are not provided with Night Effects video.

MAIN FW: USABILITY AND PERFORMANCE IMPROVEMENTS
- Improves Interval Photo quality when camera is transitioning between dark and bright scenes.
- General user interface improvements.

LABS EXTENSION: BUG FIXES
- Fixed !MWAKE=2 booting without battery (some source may require !MTUSB=1)
- Fixed bricking bug when a user error sets SPED to be permanent
- Fixed the exposure error when mxing WIDE and WBLK features
- Fixed NR01=1 so that the noise reduction is fully off
- Fixed BITL to support Kb/s (oMBITL=1500) and Mb/s (OMBITL=5)
- Fixed 24HZ=1 not seting the sensor to 24.0Hz
- Fixed w++ and w– white up and down control

Download from https://gopro.github.io/labs/

**Comment by esiremalaysia** [comment]
Great work David and Labs team !

**Comment by businessman9** [comment]
Overlay is not working on my 12.

**Reply by dnewman-gpsw** [reply]
There is a minor bug, set you font size to 30 or more. At 20 is currently not rendering, this will be fixed soon.

**Reply by kouyoumin** [reply]
Same on H11 2.20.70, setting size to 30 works.
